esmvalcore/preprocessor/_io.py

```diff
@@ -35,6 +35,8 @@
     'alternative_dataset',
 }
 
+iris.FUTURE.save_split_attrs = True
```

Comment:
luckily this is the only change in code (apart from environment changes/pins) so this is the one that's failing that test :grin: